scripts/spm-integration-test.sh

```diff
@@ -2,8 +2,34 @@
 
 set -euf -o pipefail
 
+print_help() {
+  echo "Usage: $0 [-b binary]"
+  echo "-b: Which binary to build: 'all-in-one' (default) or 'jaeger' (v2)"
+  echo "-h: Print help"
+  exit 1
+}
+
+BINARY='all-in-one'
 compose_file=docker-compose/monitor/docker-compose.yml
-timeout=300
+
+while getopts "b:h" opt; do
+	case "${opt}" in
+	b)
+		BINARY=${OPTARG}
+		;;
+	*)
+		print_help
+		;;
+	esac
+done
+
+set -x
```

Comment:
I think we can remove this now. It will clutter the CI.

Reply:
When ci is successful it doesn't matter, but when it falls it's useful to see.

Reply:
Okay.